unattended_installer/install_functions/installCommon.sh

```diff
@@ -273,17 +318,19 @@ function installCommon_installPrerequisites() {
 
 function installCommon_readPasswordFileUsers() {
 
-    filecorrect=$(grep -Ev '^#|^\s*$' "${p_file}" | grep -Pzc "\A(\s*username:[ \t]+[\'\"]?\w+[\'\"]?\s*password:[ \t]+[\'\"]?[A-Za-z0-9.*+?]+[\'\"]?\s*)+\Z")
+    filecorrect=$(grep -Ev '^#|^\s*$' "${p_file}" | grep -Pzc "\A(\s*(indexer_username|api_username|indexer_password|api_password):[ \t]+[\'\"]?[\w.*+?-]+[\'\"]?)+\Z")
     if [[ "${filecorrect}" -ne 1 ]]; then
         common_logger -e "The password file doesn't have a correct format or password uses invalid characters. Allowed characters: A-Za-z0-9.*+?
 
+It must have this format:
+
 # Description
-  username: name
-  password: password
+  indexer_username: name
+  indexer_password: password
 
-# Wazuh indexer admin user
-  username: kibanaserver
-  password: NiwXQw82pIf0dToiwczduLBnUPEvg7T0
+# Description
+  api_username: kibanaserver
+  api_password: NiwXQw82pIf0dToiwczduLBnUPEvg7T0
```

Comment:
Why? This is a help message not related to indexer

Reply:
This comment indicates the correct format for the file it should show the two options in the new format.

Reply:
Messages improved in [8d36492](https://github.com/wazuh/wazuh-packages/pull/1663/commits/8d36492b436811733ca8766e0cd6ec68929fb46f)